packages/api-client/src/components/OpenApiClientButton.vue

```diff
@@ -3,37 +3,37 @@ import { ScalarIcon } from '@scalar/components'
 import { makeUrlAbsolute } from '@scalar/oas-utils/helpers'
 import { computed } from 'vue'
 
-const { integration, isDevelopment, url, buttonSource } = defineProps<{
```

Comment:
I'm pretty sure this [breaks reactivity](https://vuejs.org/guide/extras/reactivity-transform#reactive-props-destructure).